app/services/subject_mapper.rb

```diff
@@ -134,62 +102,101 @@ def to_s
     end
   end
 
-  def self.map_to_secondary_subjects(course_title, ucas_subjects)
-    secondary_subject_mappings = MAPPINGS[:secondary].map do |ucas_input_subjects, dfe_subject|
-      GroupedSubjectMapping.new(ucas_input_subjects, dfe_subject)
+  #  Does the subject list mention languages but hasn't already been covered?
+  class MFLOtherMapping
```

Comment:
Can we keep it at one class per file? Also, would be nice to avoid jargon like `MFL` - at least it's very ambiguous to me what it means.